### ISSUE TITLE
Staging Review finding: Sort options unclear, potentially not relevant

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#126940]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** Comparison Tool
**Feature:** Student feedback grid

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility), @kkitagawa-bosch (content)

### Description

The options in the `Sort by` dropdown are `a-z` and `z-a`. Couple of issues here:

1. Unclear labels: it's not clear what is being sorted alphabetically - the name of the type of feedback/complaint? With labels that only say `a-z` and `z-a` it is not clear.

2. Type of sort: It seems unlikely that users would want to sort alphabetically by type of complaint especially given that they can filter on this information. Does user research indicate folks would want to sort by complaint type? What if there is more than one complaint type? Is there a possibility to sort by date instead?

<details><summary>Link, screenshot or steps to recreate</summary>

to be added

</details>

## Recommended action

Add a sort by date if possible

If keeping the alphabetical sort, clearly state what is being sorted alphabetically.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** [Search results template](https://design.va.gov/templates/search-results)
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
